app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -72,6 +72,20 @@
     </li>
   </ul>
 
+  <% if decorated_session.sp_name %>
+    <%= render(
+      'shared/troubleshooting_options',
+      heading_level: :h3,
```

Comment:
we only use the partial once, so why not default the partial to `h3` and remove the option?

Reply:
This current singular usage will end up being the exception, where in future pull requests the anticipated usage on the error screens it will always otherwise be H2, and I think H2 would generally be a safe default.

Reply:
ok sounds good!